app/models/document.rb

```diff
@@ -169,11 +169,12 @@ class NotArchiveableError < StandardError; end
   scope :for_publication, -> { active.publishable }
   scope :for_display, -> { active.referenced_in_decision_notice }
 
-  scope :with_tag, ->(tag) { where("tags @> ?", "\"#{tag}\"") }
+  scope :with_tag, ->(tag) { where("? = ANY(tags)", tag) }
+  scope :with_plan_tags, -> { where(tags: [PLAN_TAGS]) }
```

Comment:
This doesn't seem like it would match anything as it would require `tags` to contain all of `PLAN_TAGS` - do you mean an overlap? e.g. tags matches any of `PLAN_TAG` ?

If so you can use `overlaps` similar to `contains`, e.g.

```
scope :with_plan_tags, -> { where(arel_table[:tags].overlaps(PLAN_TAGS)) }
```